src/web/src/components/AdvancedSearchDialog.tsx

```diff
@@ -0,0 +1,188 @@
+import useSearchValue from '../hooks/use-search-value';
+import CloseIcon from '@material-ui/icons/Close';
+import { Dialog, DialogContent, DialogTitle, IconButton } from '@material-ui/core';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import React, { useState } from 'react';
+
+interface Props {
+  setOpenDialog: Function;
+  openDialog: boolean;
+  searchTerm: string;
```

Comment:
Same issue here, if this is just the search text, you can just use `text` from `useSearchValue()`